iceberg/iceberg-handler/src/main/java/org/apache/iceberg/mr/hive/HiveIcebergMetaHook.java

```diff
@@ -256,4 +265,74 @@ private static PartitionSpec spec(Schema schema, Properties properties,
       return PartitionSpec.unpartitioned();
     }
   }
+
+  @Override
+  public void commitInsertTable(org.apache.hadoop.hive.metastore.api.Table table, boolean overwrite)
+      throws MetaException {
+    String tableName = TableIdentifier.of(table.getDbName(), table.getTableName()).toString();
+
+    // check status to determine whether we need to commit or to abort
+    JobConf jobConf = new JobConf(conf);
+    String queryIdKey = jobConf.get("hive.query.id") + "." + tableName + ".result";
```

Comment:
Could we use constants here?

Reply:
Maybe even "hive.query.id.%.result"? Or "hive.query.id.result."+tableName?

Reply:
Makes sense. I was thinking of replacing the query id with a constant like `HIVE_TEZ_COMMIT_JOB_RESULT` so it would be `HIVE_TEZ_COMMIT_JOB_ID + "." + tableName`, like for job id and task num to make things consisent.